styles.css

```diff
@@ -330,3 +330,81 @@ a:hover {
 .same img {
   margin-right: 5%;
 }
+
+.contacts {
+  background-image: url('./img/aboutbackground.svg');
+  background-repeat: no-repeat;
+  display: block;
+}
+
+.contact-me {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 5%;
+}
+
+.contacts p {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
```

Comment:
[OPTIONAL]

- [ ]  The styling used in this section is a great implementation 👏🏽  You could also save some lines of code if you like, by using the shorthand form for these font properties. E.g: 
```
.contacts p {
  font: normal 20px/28px Poppins, sans-serif;
}
```